docs/content/how-to/disconnected/known-issues.md

```diff
@@ -44,3 +45,148 @@ data:
 ```
 
 - **The HO was deployed using the Hypershift binary:** In this case you will just need to remove the flag `--enable-uwm-telemetry-remote-write` from the hypershift deployment command.
+
+## HCP CLI failing to create an hosted cluster with `failed to extract release metadata: failed to get repo setup: failed to create repository client for <your.internal.registry>
+
+See: [CNV-38194](https://issues.redhat.com/browse/CNV-38194)
+Currently the HCP client is directly (client side) trying to access the OCP release image, this implies some additional requirements on client side:
+
+1. the HCP CLI client should be able to directly reach the internal mirror registry, and this could require additional configuration (a proxied access to the api server of the management server is not enough)
+1. the HCP CLI client should explicitly consume the pull secret of the internal mirror registry
+1. the HCP CLI client should explicitly consume the internal CA used to sign the TLS cert of the internal mirror registry
+
+**Workaround:**
```

Comment:
Another workaround could be set the annotation on the HostedCluster, to override the control-plane-image. Am I right? Could we add this as an additional workaround?

Reply:
No, for sure that will not be enough.
Since the enviroment is disconnected we cannot expect it to inspect the OCP release API to dynamically fetch a good one.
So a releaseImage CLI parameter should be specified and, since `getReleaseSemanticVersion` is not relying on ICSP/IDMS it should be already as on the mirror registry.
The issue is in https://github.com/openshift/hypershift/blob/671965416815480e0dc28ec4191d16b32c36583c/cmd/cluster/core/create.go#L544C6-L552

where `defaultNetworkType` is calling `getReleaseSemanticVersion` if `opts.ReleaseImage != ""`.
Now, beter looking at the code, I see that explicitly setting `--network-type OVNKubernetes` is a valid workaround.
I'll track it.